### PR TITLE
Upgrade: Add v1.0.3 Known issues

### PR DESCRIPTION
**Pull request by bk201**
Upgrade:
- Add known issues.
- Move older versions into a sub-tree.
- Add v1.0.3 support matrix

Related issues: https://github.com/harvester/harvester/issues/2053#

[Rendered](https://github.com/bk201/docs-1/blob/workaround-2053/docs/upgrade/v1-0-2-to-v1-0-3.md)

### CODE REVIEW

**Review by Vicente-Cheng — APPROVED** [review]
LGTM!

File: docs/upgrade/v1-0-2-to-v1-0-3.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    ```
+    
+    Example output:
+    
+    ```
+    NAME                              READY   STATUS    RESTARTS   AGE
+    backing-image-manager-c00e-3830   1/1     Running   0          3d1h
+    ```
+    
+- Restart the backing-image-manager:
```
Comment (guangbochen) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:209]:
```suggestion
- Restart the backing-image-manager by deleting its pod:
```

File: docs/upgrade/v1-0-2-to-v1-0-3.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    }
+    {
+      "replica": "pvc-9a40e5b9-543a-4c90-aafd-ac78b05d7599-r-bc6f7fa5",
+      "healthyAt": "",
+      "nodeID": "harvester-x9jqw",
+      "state": "stopped"
+    }
+    ```
+    
+    Here the only healthy replica is `pvc-9a40e5b9-543a-4c90-aafd-ac78b05d7599-r-15e31246` and it’s on node `harvester-z7j2g`. So we can confirm the instance-manager-r pod resides on node `harvester-z7j2g` and avoids the drain.
```
Comment (guangbochen) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:123]:
```suggestion
    Here the only healthy replica is `pvc-9a40e5b9-543a-4c90-aafd-ac78b05d7599-r-15e31246` and it’s on node `harvester-z7j2g`. So we can confirm the `instance-manager-r-*` pod resides on node `harvester-z7j2g` and avoids the drain.
```

File: docs/upgrade/v1-0-2-to-v1-0-3.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+### 2. Upgrade is stuck, a node is in "Pre-drained" state
+
+**Description**
+
+Users might see a node is stuck at the Pre-drained state for a while (> 30 minutes).
+
+![](./assets/known_issues/2053-stuck.png){: style="width:50%"}
+
+
+This might be caused by instance-manager-r pod on node `harvester-z7j2g` can’t be drained. To verify:
```
[the remark this answers is not in the document]
Comment (bk201) [reply docs/upgrade/v1-0-2-to-v1-0-3.md]: Thanks, apply all suggestions!

**Review by guangbochen — APPROVED** [review]
LGTM, and will merge after the v1.0.3 release, thanks.

**Comment by bk201** [comment]
@guangbochen @Vicente-Cheng, I add one more known issue: https://github.com/harvester/harvester/issues/2640

**Review by guangbochen — APPROVED** [review]

File: docs/upgrade/v1-0-2-to-v1-0-3.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+
+**Related issues**
+
+- [[BUG] Upgrade is stuck in "Pre-drained" state (Volume has too many system snapshots)](https://github.com/harvester/harvester/issues/2640)
+
+
+**Workaround**
+
+- In the "Snapshots and Backup" pane, toggle the "Show System Hidden" switch and delete the latest system snapshot (which is just before the "Volume Head").
```
Comment (Vicente-Cheng) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:244]: s/pane/panel

File: docs/upgrade/v1-0-2-to-v1-0-3.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+
+### 3. An upgrade is stuck, a node is in "Pre-drained" state (case 2)
+
+**Description**
+
+Users might see a node is stuck at the Pre-drained state for a while (> 30 minutes).
+
+![](./assets/known_issues/2640-stuck.png){: style="width:50%"}
+
+The following are some steps to verify this issue happens:
```
Comment (Vicente-Cheng) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:226]: `Here are some steps to verify this issue has happened:` might be better?

File: docs/upgrade/v1-0-2-to-v1-0-3.md [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+- [[BUG] Upgrade is stuck in "Pre-drained" state (Volume has too many system snapshots)](https://github.com/harvester/harvester/issues/2640)
+
+
+**Workaround**
+
+- In the "Snapshots and Backup" pane, toggle the "Show System Hidden" switch and delete the latest system snapshot (which is just before the "Volume Head").
+
+    ![](./assets/known_issues/2640-vol-3.png)
+
+- The volume will continue rebuilding and the upgrade will continue.
```
Comment (Vicente-Cheng) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:248]: `The volume will continue rebuilding to resume the upgrade.` might be better?

Comment (Vicente-Cheng) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:248]:
```suggestion
- The volume will continue rebuilding to resume the upgrade.
```

File: docs/upgrade/v1-0-2-to-v1-0-3.md (hunk 4 above)
Comment (Vicente-Cheng) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:244]:
```suggestion
- In the "Snapshots and Backup" panel, toggle the "Show System Hidden" switch and delete the latest system snapshot (which is just before the "Volume Head").
```

File: docs/upgrade/v1-0-2-to-v1-0-3.md (hunk 5 above)
Comment (Vicente-Cheng) [inline docs/upgrade/v1-0-2-to-v1-0-3.md:226]:
```suggestion
Here are some steps to verify this issue has happened:
```

**Review by Vicente-Cheng — APPROVED** [review]

**Pull request merged**